### ISSUE TITLE
Permit hidding attributes based on model attributes

### ISSUE THREAD

**Issue by k8n** [salomoni/authorized-attributes#1]
As far as I can tell, policy methods [receive the model class as a parameter](https://github.com/salomoni/authorized-attributes/blob/8fa740b04a04ba747697c6ea861b1b417bdf22d2/src/AuthorizedAttributes.php#L25).

Passing the model instance itself to the policy methods would enable hiding attributes dynamically, e.g.:
```
    public function seeAuthorComments(User $user, Post $post)
    {
        return $user->isAuthor() || $post->is_public;
    } 
```

**Comment by salomoni** [comment]
You are right, need to this.

**Comment by salomoni** [comment]
Thanks for the PR! 💯

**Comment by salomoni** [comment]
The change is included in version 1.0.2
